h/views/account_signup.py

```diff
@@ -29,21 +29,21 @@ def __init__(self, request):
             css_class="js-disable-on-submit",
         )
 
-    @view_config(request_method="GET")
+    @view_config(
+        request_method="GET", renderer="h:templates/accounts/signup.html.jinja2"
+    )
     def get(self):
         """Render the empty registration form."""
         self._redirect_if_logged_in()
 
         return {"form": self.form.render()}
 
-    @view_config(request_method="POST")
+    @view_config(
+        request_method="POST",
+        renderer="h:templates/accounts/signup-successful.html.jinja2",
```

Comment:
This is a weird name for this template, as in many cases the sign-up was not successful.

Reply:
Renamed to `signup-post.html.jinja2`: https://github.com/hypothesis/h/pull/5964/commits/a45d29b07a7f4330741269b0a769f12b49832a77